addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -471,10 +472,12 @@ PropertiesField.components = {
 PropertiesField.props = {
     ...standardFieldProps,
     columns: { type: Number, optional: true },
+    hideKanbanOption: { type: Boolean, optional: true },
 };
 PropertiesField.extractProps = ({ attrs, field }) => {
     const columns = parseInt(attrs.columns || "1");
-    return { columns };
+    const hideKanbanOption = !!parseInt(attrs.hideKanbanOption || "0");
```

Comment:
I see there is a "archParseBoolean" util method, would probably be better than using a parseInt?